### ISSUE TITLE
Infinite Scroll button (span) cannot be focused (accessibility issue)

### ISSUE THREAD

**Issue by mtomas7** [Automattic/jetpack#1205]
This is accessibility question. Recently I was testing Infinite Scroll with footer widgets and noticed that "Older posts" button (span) cannot be focused with keyboard. Is it possible to fix it?

Also it would be better to change span to button type.

Thank you!

**Comment by zinigor** [comment]
Whoops, looks like @samhotchkiss has merged this already. I have created a new issue to further improve the IS handle: #1523
Sorry to have kept you waiting @davidakennedy, but the good news is that you can test this on the `master` branch :smile:
